programmers/로또의-최고-순위와-최저-순위.js

```diff
@@ -0,0 +1,28 @@
+function solution(lottos, win_nums) {
+  const lottoGrade = [6, 6, 5, 4, 3, 2, 1];
+  const myGrades = [];
+  const answer = [];
+  let zeroCount = 0;
+  let correctAnswerCount = 0;
+
+  lottos.forEach((myNumber) => {
+    win_nums.forEach((winNumber) => {
+      if (myNumber === winNumber) {
+        correctAnswerCount += 1;
+      }
+    });
+    if (myNumber === 0) {
+      zeroCount += 1;
+    }
+  });
+
+  myGrades.push(lottoGrade[correctAnswerCount]);
+  if (zeroCount) {
+    for (let zero = 0; zero < zeroCount + 1; zero++) {
+      myGrades.push(lottoGrade[correctAnswerCount]);
+      correctAnswerCount += 1;
+    }
+  }
+  answer.push(Math.min(...myGrades), Math.max(...myGrades));
+  return answer;
```

Comment:
굳이 push하는 과정을 거칠 필요가 없습니다.
answer 변수도 필요가 없죠

함수의 결과를 return해주는 변수를 선언하는 것과 바로 값을 return 하는 것 중 어느것이 더 나은 방식인지는 솔직히 잘 모르겠네요. 하지만 이런 방식도 있으니 알아두세요 ㅋㅋ

```suggestion
  return [Math.min(...myGrades), Math.max(...myGrades)];
```